scripts/benchmarks/benches/base.py

```diff
@@ -16,12 +16,24 @@
 class Benchmark:
     def __init__(self, directory):
         self.directory = directory
-        self.adapter_path = os.path.join(options.ur_dir, 'build', 'lib', f"libur_adapter_{options.ur_adapter_name}.so")
+        for libs_dir_name in ['lib', 'lib64']:
```

Comment:
I think the make install fix is fine, but, in this case, not doing the install makes this deterministic - the file will be located in the lib directory.
Why the change to do a `make install`?

Reply:
can you make 'get lib path' into a generic function?

Reply:
> I think the make install fix is fine, but, in this case, not doing the install makes this deterministic - the file will be located in the lib directory. Why the change to do a `make install`?

Not doing install will fail benchmark in an assert in the next line with message like _could not find adapter file /home/lslusarczyk/src2/ur_install/lib64/libur_adapter_level_zero.so (and in similar lib paths)_ which looks what I would like to se, that is sample full path which was searched and still being a short message

"make install" used as cleanest way of using binaries and includes of already built UR

> can you make 'get lib path' into a generic function?
yes, separate function added